repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Run.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.pipeline.datastorage.dto
+
+import java.sql.Timestamp
+
+import cromwell.pipeline.model.wrapper.{RunId, UserId}
+import play.api.libs.json.{Format, JsResult, JsValue, Json, OFormat}
+
+final case class Run(
+  runId: RunId,
+  projectId: ProjectId,
+  projectVersion: String,
+  status: String,
+  timeStart: Timestamp,
+  timeEnd: Timestamp,
```

Comment:
What `timeEnd` should `Run` have if it is not yet completed?

Reply:
Changed to Option[Instant]